data/src/main/java/com/globant/equattrocchio/data/response/Result.java

```diff
@@ -8,14 +8,14 @@
 public class Result {
     @SerializedName("images")
     @Expose
-    private List<Image> mImages = null;
+    private List<ImageResponse> mImageResponses = null;

```

Comment:
You should not use this code style, and not necessary asign to null (is default value) . Use  

private List<ImageResponse> imageResponses; 
or
private List<ImageResponse> images;

Reply:
ok, done